packages/bpk-component-ticket/src/bpk-ticket.scss

```diff
@@ -47,6 +47,7 @@
   &__paper {
     position: relative;
     background-color: $bpk-card-background-color;
+    transform-style: preserve-3d; // FSS-3894
```

Comment:
We try to keep JIRA links out of the Backpack source code, as it's open source so an external viewer won't be able to reference it. Could you replace it with a short comment explaining it's to prevent flickering in Safari? Same for line 67 too.

Reply:
Done 😀